source/simulator/core.cc

```diff
@@ -549,8 +549,8 @@ namespace aspect
   {
     // first produce some output for the screen to show where we are
     {
-      const char *unit = (parameters.convert_to_years) ? "years" : "seconds";
-      const double factor = (parameters.convert_to_years) ? 1./year_in_seconds : 1.0;
+      const char *unit = (parameters.convert_to_years ? "years" : "seconds");
```

Comment:
Whether it's an improvement might be subjective: I thought that putting parentheses around just the variable itself was probably not what you had in mind -- but putting them around the entire expression might make thinking about operator precedence easier. We often have parentheses around `?:` expressions.

Reply:
Interesting. I did have that in mind because I generally like to put the thing in front of ? in parentheses (imagine having ``!convert_to_years`` or ``convert_to_years==true``). Anyways, I don't really care either way.

Reply:
Right, if the condition is an actual expression, I totally agree. It was more a question of what to do if the variable is only a single name.